staging/src/k8s.io/apiserver/pkg/audit/policy/checker_test.go

```diff
@@ -161,7 +161,22 @@ func TestChecker(t *testing.T) {
 		require.Contains(t, attrs, req)
 		actualLevel, actualOmitStages := NewChecker(&policy).LevelAndStages(attrs[req])
 		assert.Equal(t, expLevel, actualLevel, "request:%s rules:%s", req, strings.Join(ruleNames, ","))
-		assert.Equal(t, expOmitStages, actualOmitStages, "request:%s rules:%s", req, strings.Join(ruleNames, ","))
+		assert.True(t, stageEqual(expOmitStages, actualOmitStages), "request:%s rules:%s, expected stages: %v, actual stages: %v",
+			req, strings.Join(ruleNames, ","), expOmitStages, actualOmitStages)
+
+		// test policy.OmitStages
```

Comment:
Please don't do this (hacking around existing code structure and adding a comment)

Change the `test` signature and move test cases out of the method

Reply:
Done.